.github/workflows/publish.yml

```diff
@@ -54,6 +56,8 @@ jobs:
         uses: JS-DevTools/npm-publish@v3
         with:
           token: ${{ secrets.NPM_TOKEN }}
+          provenance: true
+          access: public
```

Comment:
I think that `access: public` is not needed as is the default. Or is related by some way?

Reply:
Ah, yes. It's not necessary since it's not our first time to publish the package.